mindjunkies/courses/urls.py

```
@@ -7,6 +7,11 @@
     path("my_courses/", views.user_course_list, name="my_course_list"),
     path("create_course/", views.CreateCourseView.as_view(), name="create_course"),
     path("edit_course/", views.CourseUpdateView.as_view(), name="edit_course"),
+
+    path('create-course-token/', views.CreateCourseTokenView.as_view(), name='create_course_token'),
+
     path("<str:slug>/", views.course_details, name="course_details"),
     path("<str:course_slug>/", include("mindjunkies.lecture.urls")),
+    path("<str:course_slug>/forums/", include("mindjunkies.forums.urls")),
+    
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)